R/key-helpers.R

```diff
@@ -72,7 +71,6 @@ check_key <- function(.data, ...) {
   duplicate_rows <-
     .data %>%
     count(!!!syms(cols_chosen)) %>%
-    count(n) %>%
```

Comment:
What's the reason for this change?

Reply:
for each group we get a count of how many times it appears (n = ?). So we just need to check if `n > 1`, we don't benefit of counting how many times we get `n = 1`, if I am not mistaken

Reply:
The `count(n)` compresses what's returned from the database. We could do something like `. %>% select(n) %>% filter(n > 1) %>% head(1)` to achieve minimum overhead for both database and data frame.